tests/e2e/helper/helper_run.go

```diff
@@ -0,0 +1,40 @@
+package helper
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+)
+
+// and returns the stdout, stderr and exitcode
+func cmdRunner(cmdS string) (string, string, int) {
+	//TODO: this needs to be os independent
```

Comment:
lets document this in an issue so that @amitkrout can work on making this os independent

Reply:
@girishramnani We have issue for it #1131